runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineContext.java

```diff
@@ -143,4 +144,9 @@ void onExporterAttached(
 
     void onExporterDetached(
         long exporterId);
+
+    void logEvent(int msgTypeId,
```

Comment:
I was thinking we might want to have the `EngineContext` provide the `logger` as a `MessageConsumer`.
```
   MessageConsumer logger();
```